mpd/__init__.py

```diff
@@ -0,0 +1,215 @@
+import os
+from albertv0 import *
+from mpd import MPDClient
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "mpd: control your MPD"
+__version__ = "1.0"
+__trigger__ = "mpd "
+__author__ = "Markus Richter"
+__dependencies__ = []
+
+SERVER = "localhost"
+PORT = 6600
+
+# All included icons are part of the papirus icon theme by the Papirus Development Team and are licensed under the GPLv3.
```

Comment:
please use the packed icons only as fallback use albertv0.iconlookup() to find themed icons.

Reply:
The problem here is that some of these icons are somewhat misused. Eg. the icon representing an artist collection is an icon which usually is named "folder-home" or "user-home". As far as I'm aware, there is no canonical icon for artists. If would choose to select the "folder-home" icon of the current system icon set, there would probably pop up a folder with a house on it which would be a really bad choice for an icon representing an artist. A similar, but smaller case can be made for the "folder-music" icon representing an album... 

As for the single songs, I'll choose "audio-x-generic" which seems to be a good fit.

Reply:
As an aside: where(and how) does iconLookup look for Icons?